sdk/aqueduct/integrations/sql_integration.py

```diff
@@ -150,32 +152,59 @@ def sql(
             extract_params = RelationalDBExtractParams(
                 query=extract_params,
             )
+        elif isinstance(extract_params, list):
+            for q in extract_params:
+                assert isinstance(
+                    q, str
+                ), "When using a list of queries, it must be a list of strings."
+
+            if len(extract_params) == 1:
+                extract_params = RelationalDBExtractParams(
+                    query=extract_params[0],
+                )
+            else:
+                extract_params = RelationalDBExtractParams(
+                    queries=extract_params,
+                )
+        elif isinstance(
+            extract_params, RelationalDBExtractParams
+        ):  # query is a RelationalDBExtractParams object
+            if int(bool(extract_params.query)) + int(bool(extract_params.queries)) != 1:
+                raise Exception(
+                    "For a RelationalDBExtractParams object, exactly one of .query or .queries fields should be set."
+                )
```

Comment:
I have a slight preference for having a single field on RelationalDBExtractParams for queries with `Union[str, List[str]]` type, since that's what we've been doing throughout the SDK - see `artifacts` parameter in `publish_flow()`.